docs/app-integration-development/10-OAuth-2-PKCE.md

```diff
@@ -129,7 +129,15 @@ Authorization: Bearer pdus+_0XBPWQQ_39435d07-9232-4bc2-9dc9-c4a8fccc94ad
 Accept: application/vnd.pagerduty+json;version=2
 ```
 
-Note however, that our access tokens do expire after a defined period of time -- so you may want to make sure that you implement OAuth refresh to prevent users needing to re-authorize your app. See more information about token expiries at the bottom of this page.
```

Comment:
Did you remove this section for any reason in particular? I'm assuming we still want people to implement OAuth refresh when applicable?

Reply:
Ah: that was on purpose :laughing: sometimes people skim.